CONTRIBUTING.md

```diff
@@ -0,0 +1,36 @@
+# 维护指南
+
+欢迎地震学常用软件的用户参与到教程的维护与更新中。
+
+## 维护与更新
+
+教程的维护与更性主要包括几个方面：
+
+- 修正错别字、语句不通等
+- 修正教程中的错误或不清晰的描述
+- 修正reST文件语法错误导致的显示问题
+- 调整章节结构，使得教程条理更清晰
+- 建议、编写或者整理其他地震学常用软件的教程
+
+可以通过如下几种方式参与教程的维护：
+
+1. 通过 GitHub 修改教程源码并提交 Pull Request
+2. 在 GitHub 上提交 Issue
+3. 发送相关建议或投稿至 `seisman.info@gmail.com` | `coreman.seism@gmail.com`
```

Comment:
Add this one https://github.com/seismo-learn/software/discussions?

Reply:
Shall we remove emails?

Reply:
I think so.